com.stansassets.scene-management/Editor/Window/Tabs/SettingsTab.uxml

```diff
@@ -1,9 +1,15 @@
-<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" xmlns:sa="StansAssets.Foundation.UIElements">
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" 
+         xmlns:sa="StansAssets.Foundation.UIElements" editor-extension-mode="True">
+    
     <sa:SettingsBlock label="Optional Settings">
-        <uie:ObjectField label="Landing Scene" name="landing-scene" />
+        <uie:ObjectField label="Landing Scene" name="landing-scene"/>
+        <ui:VisualElement name="scene-persistence" style="height: auto;">
+            <uie:EnumField label="Use Scene And Camera Persistence" name="persistence-enum-field"/>
```

Comment:
I think it would be better to rename this option to Scene & Camera Persistence.
```suggestion
            <uie:EnumField label="Scene & Camera Persistence" name="persistence-enum-field"/>
```